sources/packages/web/src/services/http/BypassedRestrictionApi.ts

```diff
@@ -0,0 +1,22 @@
+import HttpBaseClient from "@/services/http/common/HttpBaseClient";
+import { BypassedRestrictionSummaryAPIOutDTO } from "@/services/http/dto";
+
+/**
+ * Http API client for Bypassed Restrictions.
+ */
+export class BypassedRestrictionApi extends HttpBaseClient {
```

Comment:
class name `ApplicationRestrictionBypassApi`? and file name likewise?

Reply:
Updated.